patches/11-openshiftv1-cli.patch

```diff
@@ -1,43 +1,35 @@
 diff -up ./internal/cmd/operator-sdk/cli/cli.go.patchocpv1 ./internal/cmd/operator-sdk/cli/cli.go
---- ./internal/cmd/operator-sdk/cli/cli.go.patchocpv1	2024-07-25 19:58:43.805980081 +0530
-+++ ./internal/cmd/operator-sdk/cli/cli.go	2024-07-25 20:03:21.138415443 +0530
-@@ -52,6 +52,7 @@ import (
- 	envtestv1 "github.com/operator-framework/operator-sdk/internal/plugins/envtest/v1"
+--- ./internal/cmd/operator-sdk/cli/cli.go.patchocpv1	2024-12-26 18:02:08.422724303 +0530
++++ ./internal/cmd/operator-sdk/cli/cli.go	2024-12-26 18:04:50.221965067 +0530
+@@ -42,6 +42,7 @@ import (
+ 	"github.com/operator-framework/operator-sdk/internal/plugins"
  	helmv1 "github.com/operator-framework/operator-sdk/internal/plugins/helm/v1"
  	manifestsv2 "github.com/operator-framework/operator-sdk/internal/plugins/manifests/v2"
 +	openshiftv1 "github.com/operator-framework/operator-sdk/internal/plugins/openshift/v1"
```

Comment:
What exactly is the fix here?

Reply:
The openshift plugin is added to the downstream osdk cli. This patch file adds it. However the cli.go file was updated due to the removal of the hybrid-helm plugin and quarkus plugin. Thus, this patch file needs to be regenerated.

Reply:
Ack, thanks for the explanation.